src/components/Navbar/components/CurrentLocationWidget/CurrentLocationWidget.tsx

```diff
@@ -0,0 +1,71 @@
+import ContentLoader from 'react-content-loader';
+import { Weather } from 'models/Weather';
+import { Container, WeatherIcon, Wrapper, Description, DegreeSign } from './CurrentLocationWidget.styled';
+
+const CurrentLocationWidget = () => {
+  let isLoading = false;
+  const data: Weather | any = {}; // temporary solution in order not to use
+
+  const renderContent = () => {
+    if (isLoading) {
+      if (screen.width > 600) {
```

Comment:
**suggestion** I've noticed that `screen` doesn't work when user is resizing window screen.
Maybe consider making some active classes for those loaders :)

Reply:
What do you mean by active classes?

Reply:
I mean classes that changes it's display styles when viewport is e.g. > 600px.
E.g. You have two loaders with two different classes: `mobile` and `tablet`.
And when 
```
media screen and (max-width: 600px) {
    .tablet {
        display: none;
    }
    .mobile {
        display: block;
    }
}
```

This is just an example. I'm just thinking how to make it in css ;)

Reply:
Okay, I will try to think about this sort of solution

Reply:
I am also thinking what is the best solution for that :D
Maybe @kamil-borecki-cc @bednarskimichal  or @dkaczmarek-cc  can tell something about it :)

Reply:
We want to handle RWD with CSS not with js

Reply:
I am trying go with the code: 

.current-location-widget-desktop {
    display: inline-block;
  }

  .current-location-widget-mobile {
    display: none;
  }

  @media screen and (max-width: '600px') {
    .current-location-widget-desktop {
      display: none;
    }

    .current-location-widget-mobile {
      display: inline-block;
    }
  }

and the loaders are as given

 <>
          <div className="current-location-widget-desktop">
            <CurrentLocationWidgetDesktop />
          </div>
          <div className="current-location-widget-mobile">
            <CurrentLocationWidgetMobile />
          </div>
</>

But it doesn't work as expected